client/containers/Pub/PubDocument/PubBottom/Notes.tsx

```diff
@@ -74,13 +68,12 @@ const Note = (props: NoteProps) => {
 
 	return (
 		<li className="note">
-			{/* @ts-expect-error ts-migrate(2339) FIXME: Property 'number' does not exist on type 'noteProp... Remove this comment to see the full error message */}
-			<div className="number">{note.number}.</div>
+			{note.number && <div className="number">{note.number}.</div>}
 			<div className="inner">
 				<PubNoteContent
 					ref={contentRef}
-					structured={citation && citation.html}
-					unstructured={note.unstructuredValue}
+					structured={linkifyHtml(note.renderedStructuredValue?.html)}
```

Comment:
❗️ I think it's likely that we want the call to `linkifyHtml` to live inside the `PubNoteContent` component, so this behavior will get picked up in the `ControlsFootnoteCitation` as well. Might be worth wrapping these computations in `useMemo` as well.